src/storage.py

```diff
@@ -223,24 +231,27 @@ def remove_osd(self, osd_num: int, force: bool = False):
 
     def _save_osd_data(self, disk_name: str, db_name: str = None):
         """Save OSD data using juju storage names."""
```

Comment:
It would be great if the docstring would elaborate on the semantics of this function.

Also, it looks like the db_name param is not actually being used? (I realize that this is not part of the current change)

Reply:
ack